src/Core/src/Core-net6.csproj

```diff
@@ -19,6 +19,7 @@
     <PackageReference Include="Microsoft.Extensions.Configuration" />
     <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" />
+		<PackageReference Include="Microsoft.Maui.Graphics" Version="6.0.100-preview.3.29" />
```

Comment:
Can we use the same pattern here, too? Maybe an `eng/Microsoft.Maui.Graphics.targets` file with:
```
<PackageReference Update="Microsoft.Maui.Graphics" Version="6.0.100-preview.3.29" />
```
You would need to import the new `.targets` file here:
https://github.com/dotnet/maui/blob/e0b7d31cc600a9a2918489bb57cc2ae00c8e9d76/Directory.Build.targets#L4-L5